mne/preprocessing/tests/test_ica.py

```diff
@@ -651,7 +656,7 @@ def f(x, y):
     idx, scores = ica.find_bads_eog(evoked, ch_name='MEG 1441')
     assert_equal(len(scores), ica.n_components_)
 
-    idx, scores = ica.find_bads_ecg(evoked, method='correlation')
+    idx, scores = ica.find_bads_ecg(evoked, method='correlation', threshold=3)
```

Comment:
Can this be changed to `threshold='auto'` and still work?

Reply:
This has not been checked yet

Reply:
Thanks for reminding, I've checked and updated the influenced tests and examples.